index.bs

```diff
@@ -11215,14 +11232,17 @@ in which case they are exposed on every object that [=implements=] the interface
     To <dfn>define the operations</dfn> |operations| of [=interface=] or [=namespace=] |definition| on |target|,
     given [=Realm=] |realm|, run the following steps:
 
+    1.  Let |enumerable| be <emu-val>false</emu-val> if |definition| is an [=interface=] that has
+        the [{{UnenumerableOperations}}] [=extended attribute=] and <emu-val>true</emu-val>
+        otherwise.
```

Comment:
This&nbsp;should also&nbsp;be&nbsp;defined for&nbsp;[**Stringifiers**](https://heycam.github.io/webidl/#es-stringifier), [**Common&nbsp;iterator&nbsp;behavior** §&nbsp;`forEach`](https://heycam.github.io/webidl/#es-forEach) and&nbsp;[**Iterable&nbsp;declarations**](https://heycam.github.io/webidl/#es-iterable), like&nbsp;I’m&nbsp;doing&nbsp;in&nbsp;<https://github.com/heycam/webidl/pull/825>.